src/caret_analyze/value_objects/callback.py

```diff
@@ -820,10 +856,12 @@ def __init__(
             node_name=node_name,
             symbol=symbol,
             subscribe_topic_name=subscribe_topic_name,
+            subscription_construction_order=subscription_construction_order,
             service_name=None,
-            publish_topic_names=publish_topic_names,
+            publish_topics=publish_topics,
             construction_order=construction_order,
             callback_name=callback_name)
+        self._subscription_construction_order = subscription_construction_order
```

Comment:
Is this line really necessary?

Reply:
subscription_construction_order itself seems unnecessary, so I deleted it.